server/index.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const path = require('path');
+const bodyParser = require('body-parser');
+const routes = require('./routes.js');
+
+const app = express();
+const port = 3000;
+
+app.use(bodyParser.urlencoded({ extended: false }));
```

Comment:
I always do extended: true, but not sure if it makes a difference.

Reply:
Updated.